frontend/src/components/Navbar.jsx

```diff
@@ -1,12 +1,13 @@
-import React from "react";
-import { useContext } from "react";
-import TextField from "@mui/material/TextField";
-import Button from "react-bootstrap/Button";
-import Logo from "./my_unsplash_logo.svg";
+import { TextField } from "@mui/material";
+import React, { useContext } from "react";
 import MainContext from "../context/MainContext";
+import AddPhotoModal from "./AddPhotoModal";
+import Logo from "./my_unsplash_logo.svg";
 
 function Navbar() {
   const { search, setSearch } = useContext(MainContext);
+  const handleSubmit = (photoLabel, photoUrl) =>
+    console.log(`Label: ${photoLabel}; URL: ${photoUrl}`);
```

Comment:
The log appear for a brief second before disappearing. But I believe it's just the way `onClick` events are handled. When backend REST API is done, we will use photoLabel and photoURL as parameters for the PhotoModel to be created.